File: chapter4/file_pointers/phonebook.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Open CSV (comma separated values) file
    FILE *fonebook = fopen("phonebook.csv", "a");

    // Stop the program if the file doesn't open
    if (!fonebook)
    {
        return 1;
    }

    // Get name and number
    char *name = get_string("Name: ");
    char *number = get_string("Number: ");

    // Print to file
    fprintf(fonebook, "%s,%s\n", name, number);

    // Close file
    fclose(fonebook);

}

File: chapter5/csv/phonebook.py
import csv
from cs50 import get_string

# Open phonbebook.csv in "a"ppend mode
file = open("phonebook.csv", "a")

# Get number from the user
name = get_string("Name: ")
number = get_string("Number: ")

# Write the name and number to the file
writer = csv.writer(file)
writer.writerow([name, number])

# Close the open file
file.close()
